geowebcache/kml/src/main/java/org/geowebcache/service/kml/KMLSiteMap.java

```diff
@@ -37,12 +38,14 @@
     private TileLayerDispatcher tLD = null;
     private StorageBroker storageBroker;
     private GridSetBroker gridSetBroker;
+    private SecurityDispatcher secDispatcher;
     
-    public KMLSiteMap(ConveyorKMLTile tile, TileLayerDispatcher tLD, GridSetBroker gridSetBroker) {
+    public KMLSiteMap(ConveyorKMLTile tile, TileLayerDispatcher tLD, GridSetBroker gridSetBroker, SecurityDispatcher secDispatcher) {
```

Comment:
GeoWebCacheDispatcher and KMLService both use setSecurityDispacher. This takes it in as an argument. Any reason for the difference?

Reply:
For the Spring beans (Services etc) I started with properties, but this is instantiated normally which made it more convenient to make it a constructor argument.